data/en/getsoaprequest.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name":"getSOAPRequest",
 	"type":"function",
-	"syntax":"getSOAPRequest()",
+	"syntax":"getSOAPRequest(webservice)",
```

Comment:
Good catch here, this is good, however we also need to add it to the `params` array, like this:
```
"params": [
		{"name":"webservice","description":"The description.","required":false,"default":"","type":"string","values":[]}
],

Note: I didn't fill in a `description` above
```

Reply:
Should I also change the type from "string" to "any" as it's listed in the Lucee docs?

Reply:
yes, should be `any` - good catch